common/static/coffee/spec/discussion/view/discussion_thread_view_spec.coffee

```diff
@@ -61,8 +61,18 @@ describe "DiscussionThreadView", ->
               (model, updates, safeAjaxParams, errorMsg) ->
                 model.set(updates)
             )
+
             view
 
+        checkForTopicFocus = () ->
+            view = createDiscussionThreadView()
+            waitsFor (->
+                $(view.$(".discussion-article"))
+            ), "conversation did not receive focus", 5000
+
+        it "sends focus to the conversation when opened", ->
+            checkForTopicFocus()
```

Comment:
This test point succeeds with or without your code change in discussion_thread_view.coffee.

Your condition is just checking that an element is "truthy"-- as far as I can tell, it's not actually verifying anything about focus.